roles/vm/tasks/main.yml

```diff
@@ -18,7 +18,7 @@
     node: "{{ item.node }}"
     scsi: "{{ item.scsi | default(omit) }}"
     scsihw: "{{ item.scsihw | default(omit) }}"
-    update: "{{ item.update | default(omit) }}"
+    update: "{{ item.refresh | default(omit) }}"  # do NOT use item.update ! Seen as python method from ansible POV
```

Comment:
Why don't you use `item['update']` instead?

If you want to continue with `item.refresh`, please remove the comment from the code and explain the change in the commit message instead.

Reply:
Comment removed